content/pt-br/docs/tasks/access-application-cluster/configure-access-multiple-clusters.md

```diff
@@ -0,0 +1,400 @@
+
+{{< note >}}
+Um arquivo usado para configurar o acesso a um cluster às vezes é chamado de arquivo *kubeconfig*. Essa é uma forma genérica de se referir a arquivos de configuração. Isso não significa que exista um arquivo chamado `kubeconfig`.
+{{< /note >}}
+
+
+{{< warning >}}
+Use somente arquivos kubeconfig de fontes confiáveis. O uso de um arquivo kubeconfig especialmente criado, pode resultar em execução de código malicioso ou exposição de arquivos. Se você preciso usar um arquivo kubeconfig não confiável, inspecione-o cuidadosamente antes, da mesma forma que faria com um script de shell.
+{{< /warning>}}
+
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+Para verificar se {{< glossary_tooltip text="kubectl" term_id="kubectl" >}} está instalado,
+execute `kubectl version --client`. A versão do kubectl deve ser [dentro de uma versão secundária](/releases/version-skew-policy/#kubectl) do servidor de API do seu cluster.
+
+<!-- steps -->
+
+## Defina clusters, usuários e contextos
+
+Suponha que você tenha dois clusters, um para o trabalho de desenvolvimento e outro para o trabalho de teste.
+No cluster `development`, seus desenvolvedores de front-end trabalham em um namespace chamado `frontend`,
+e os desenvolvedores de armazenamento trabalham em um namespace chamado `storage`. Em seu cluster `test`,
+os desenvolvedores trabalham no namespace padrão ou criam namespaces auxiliares conforme
+acharem adequado. O acesso ao cluster de desenvolvimento requer autenticação por certificado. O acesso
+ao cluster de teste requer autenticação por nome de usuário e senha.
+
+Crie um diretório chamado `config-exercise`. Em seu diretório
+`config-exercise`, crie um arquivo chamado `config-demo` com este conteúdo:
+
+```yaml
+apiVersion: v1
+kind: Config
+preferences: {}
+
+clusters:
+- cluster:
+  name: development
+- cluster:
+  name: test
+
+users:
+- name: developer
+- name: experimenter
+
+contexts:
+- context:
+  name: dev-frontend
+- context:
+  name: dev-storage
+- context:
+  name: exp-test
+```
+
+Um arquivo de configuração descreve clusters, usuários e contextos. Seu arquivo `config-demo
+tem a estrutura para descrever dois clusters, dois usuários e três contextos.
+
+Vá para o diretório `config-exercise`. Digite estes comandos para adicionar detalhes do cluster ao
+seu arquivo de configuração:
+
+```shell
+kubectl config --kubeconfig=config-demo set-cluster development --server=https://1.2.3.4 --certificate-authority=fake-ca-file
+kubectl config --kubeconfig=config-demo set-cluster test --server=https://5.6.7.8 --insecure-skip-tls-verify
+```
+
+Adicione detalhes do usuário ao seu arquivo de configuração:
+
+{{< caution >}}
+O salvamento de senhas na configuração do cliente do Kubernetes é arriscado. Uma alternativa melhor seria usar um plug-in de credenciais e salvá-las separadamente. Veja: [client-go credential plugins](/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins)
```

Comment:
Small suggestion for this paragraph and a correction for the link:

```suggestion
O armazenamento de senhas na configuração do cliente do Kubernetes é arriscado. Uma alternativa melhor seria usar um plug-in de credenciais e salvá-las separadamente. Veja: [plugins de credenciais client-go](/pt-br/docs/reference/access-authn-authz/authentication/#plugins-de-credenciais-client-go)
```

Reply:
Armazenamento does sound much better than salvamento. Done.